src/redux/books/booksSlice.js

```diff
@@ -1,19 +1,40 @@
 import { createSlice } from '@reduxjs/toolkit';
 
+const bookItems = [
+  {
+    item_id: 'item1',
+    title: 'The Great Gatsby',
+    author: 'John Smith',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item2',
+    title: 'Anna Karenina',
+    author: 'Leo Tolstoy',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item3',
+    title: 'The Selfish Gene',
+    author: 'Richard Dawkins',
+    category: 'Nonfiction',
+  },
+];
 const initialState = {
-  books: [],
+  books: bookItems,
 };
 
 const booksSlice = createSlice({
   name: 'books',
   initialState,
   reducers: {
     addBook: (state, action) => {
-      state.books = action.payload;
+      state.books = [...state.books, action.payload];
+      // console.log(state.books, action.payload);
```

Comment:
- [ Required ] I think it would be better if you remove any piece of commented code from your code base. Commented code makes your project unprofessional and is also considered a sign of code that is not clean. Please consider rectifying this.